packer/files/add_centos_user.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/bash
+print_usage() {
+    echo "add_centos_user.sh [--os centos/ubuntu] [--copy]"
+    exit 1
+}
+OS="centos"
+COPY=""
+while [ $# -gt 0 ]; do
+    case "$1" in
+        "--os")
+            OS="$2"
+            shift 2
+            ;;
+        "--copy")
+            COPY="1"
+            shift 1
+            ;;
+        *)
+            print_usage
+            ;;
+    esac
+done
+
+if [ "$OS" = "ubuntu" ]; then
+    sudo useradd -m -s $(which bash) -G sudo -G docker centos
```

Comment:
such a confusing name for a user...

Reply:
The whole add-centos-user logic is a terrible cloud-related-workaround that should be removed, but now the focus is on copying the existing process, later we will sort out all this spaghetti